src/Nav.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"; 
+import './Nav.css';
+
+import {
+    Link, 
+} from "react-router-dom";
+
+import {
+    Navbar,
+    NavbarBrand,
```

Comment:
unused import

(for now, tho we will probably want to add a brand at some point)

Reply:
Done.